src/compact_filter.h

```diff
@@ -64,9 +64,13 @@ class SubKeyFilter : public rocksdb::CompactionFilter {
         stor_(storage) {}
 
   const char *Name() const override { return "SubkeyFilter"; }
-  bool IsKeyExpired(const InternalKey &ikey, const Slice &value) const;
+  bool GetMetadata(const InternalKey &ikey, Metadata* metadata) const;
+  bool IsMetadataExpired(const InternalKey &ikey, const Metadata& metadata) const;
   bool Filter(int level, const Slice &key, const Slice &value,
               std::string *new_value, bool *modified) const override;
+  rocksdb::CompactionFilter::Decision FilterBlobByKey(int /*level*/, const Slice& /*key*/,
+                                   std::string* /*new_value*/,
+                                   std::string* /*skip_until*/) const override;
```

Comment:
```suggestion
  rocksdb::CompactionFilter::Decision FilterBlobByKey(int level, const Slice& key,
                                   std::string* new_value,
                                   std::string* skip_until) const override;
```

I think we should unify the style here.

Reply:
Good, thanks